### ISSUE TITLE
Domain Login Dropdown

### ISSUE THREAD

**Issue by ertjlane** [BIDMCDigitalPsychiatry/LAMP-platform#646]
**Is your feature request related to a problem? Please describe.**
mindLAMP users are required to submit a domain during login. Domains are long, complicated, and easy to confuse with each other.

**Describe the solution you'd like**
Set the domain entry in the login to dropdown a list of pertinent domains, particularly those involved with ProNet. The dropdown would also allow users to manually type in a domain.

**Comment by ertjlane** [comment]
This issue is high priority cc/@jtorous - remove from on hold

**Comment by avaidyam** [comment]
@ertjlane How do you propose we track and update this list of domains independently from the dashboard codebase?

**Comment by mcurbelo-orangeloops** [comment]
@avaidyam #632 is a PR to address this request

**Comment by avaidyam** [comment]
@mcurbelo-orangeloops I've merged your PR! (In the future, feel free to merge the PRs on your end once code review is complete.)

**Comment by ertjlane** [comment]
This issue appears to be fixed in staging!
